doc/source/conf.py

```diff
@@ -0,0 +1,67 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "nomenclature"
+copyright = "2021, IAMconsortium"
```

Comment:
I'd prefer to use the ECE SceSe team as author and IIASA as copyright holder for the time being, see https://github.com/IAMconsortium/nomenclature/blob/621fd88b73c7e03c1d7129b8ef5664f03e588540/setup.cfg#L3